### DISCUSSION TITLE
How to backup from latest version to older version 2.9.3?

### DISCUSSION THREAD

**Discussion by Nik4an** [NomicFoundation/hardhat#3265] [Troubleshooting] [answered]
I accidentally installed the wrong version of hardhat. I need another version. Deleted the node.js, but it didn't help

**Comment by fvictorio** [comment, marked as answer]
Use `"hardhat": "2.9.3"` in your `package.json`` and re-install your packages (either `npm install` or `yarn`, whatever you use)
